lms/djangoapps/oauth2_handler/tests.py

```diff
@@ -139,12 +139,12 @@ def test_permissions_scope(self):
         _scopes, claims = self.get_id_token_values('openid profile permissions')
         self.assertTrue(claims['administrator'])
 
+    @override_settings(RATELIMIT_RATE='1/m')
```

Comment:
@jmbowman this test is failing. If I execute it alone it works fine but with whole class it fails.
 This test case is doing some thing `test_no_special_course_access`. I have tried to clear cache but still test is failing. I will do debugging again tomorrow.

Reply:
@jmbowman any ideas. Now If i run the test file alone it works fine but on travis this test fails.

Reply:
@jmbowman this test was failing on travis only. So I have update the settings in tests settings.